src/components/styled-components/movie-image/styled-image.jsx

```diff
@@ -4,7 +4,8 @@ const StyledImage = styled.img`
   width: 240px;
```

Comment:
In the future when you want this to be responsive, you will find having this value static a problem. Consider using a more responsive method of measurement such as %